sci-geosciences/dcw-gmt/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
```

Comment:
I think that at this point we can assume sci-geo is not going to take it. If you'd like to proxy-maintain it, please add yourself and proxy-maint team.

Reply:
What does this mean, how to do it, and what will be my responsibilities?

Reply:
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Privileges_and_responsibilities_of_proxied_maintainers and below.

Reply:
So, I must contact the team sci-geosciences and offer myself as a proxy-maintainer?

Reply:
Nah, we've already established they don't care. Just add yourself.